### ISSUE TITLE
Flask of Petrification should give a buff, not a debuff

### ISSUE THREAD

**Issue by graciouswow** [LightsHope/issues#1269]
Flask of Petrification should give the user a buff, rather than a debuff like it currently does on your servers. Proof from 2006 of the effect is shown here: https://www.youtube.com/watch?v=kCj1FVfnPvc

Currently on your servers, it works as a debuff, and as such you can't remove it after use and have to wait out its 60 second duration. This vastly changes the potential uses for the item. For reference from your servers, see this clip https://clips.twitch.tv/SlickSteamyRavenTheThing

TL;DR: It should work like a Limited Invulnerability Potion that also makes you immune to magic. But for 60 seconds. It should also replace the current flask you have used, as the tooltip indicates.
This function isn't working properly either, as shown in the twitch stream above.

If you want to investigate it further with comments from 2005-2006, you can read them here https://web.archive.org/web/20060526120447/http://wow.allakhazam.com/db/item.html?witem=13506
